docs/user_guide/fairness_in_machine_learning.rst

```diff
@@ -141,6 +141,199 @@ which groups these values were observed, as well as the difference and ratio
+speed will fail to meaningfully address the needs of the community.
+
+As an example, the Federation of Canadian municipalities released a report
+ tackling both connectivity and infrastructure needs of rural communities to
+more meaningfully address this social issue
+https://fcm.ca/sites/default/files/documents/resources/report/broadband-access-rural-canada.pdf.
+
+The Ripple Effect Trap
+^^^^^^^^^^^^^^^^^^^^^^
+
+This trap occurs when we do not study the unintended consequences of introducing
+technology into an existing social system. Such consequences include change in
+behaviour, outcomes individual experience and a change in underlying social values
+and incentives of a given social system, for instance by increasing perceived value
+of quantifiable metrics over non-quantifiable ones.
+
+Example: consider the problem of banks deciding whether an individual should
+be approved for a loan. Prior to using machine learning algorithms
+to compute a "score", banks might rely on engaging in conversations with
+clients, recommending a plan based on their unique situation, and
+discussing with other team members to obtain feedback. By introducing an
+algorithm, it is possible that banks stop engaging in conversations
+with team members and clients, and assume the algorithm is good enough
+to blindly accept a recommendation. Furthermore, some might use
+information on people algorithm typically rejects to create a system
+that takes advantage of people who need funds but did not receive them.
+
+To avoid this pitfall, we must be aware that once a technology is incorporated
+into a social context, new groups may reinterpret it and potentially use this
+information for outside purposes which include nefarious ones. To
+solve for this issue, we must incorporate "what if" scenarios into testing
+of the model.
+
+The Formalism Trap
+^^^^^^^^^^^^^^^^^^
+
+This trap occurs when abstraction results in failure to take
+into account the full picture behind fairness in a situation. Many tasks of a data
+scientist involve some form of formalization: from measuring real-world phenomena as
+data to translating business KPI's and constraints into metrics, loss functions, or parameters.
+We fall into the formalism trap when this translation fails to account the social dimensions
+associated to the problem. Social dimensions that cannot be abstracted through a purely mathematical
+framework include procedurality, contextuality and contestability.
+
+Example: Consider the use of "risk scores" to decide the outcomes individuals
+will experience. Risk scores may be used to determine risk of recidivism in crime,
+risk of failure within a new role during recruitment, or risk of failure in a
+patient's chosen treatment. A key element within these algorithms is the
+ratio of false and true positives, as well as and false and true negatives.
+That is, how successful an algorithm is to correctly asses risk in an individual.
```

Comment:
Hmm, I'm not sure this example gets this idea across.. @hildeweerts or @romanlutz thoughts?

Reply:
Removed example for now. Can create a different issue to select better examples as needed.